File: tests/test-minecraft/src/main/java/pize/tests/minecraft/game/options/SoundCategory.java
package pize.tests.minecraft.game.options;

public enum SoundCategory{

    MASTER(1F, "audioSettings.master"), // Общая Громкость
    MUSIC(0.1F, "audioSettings.music"), // Музыка
    AMBIENT(1F, "audioSettings.ambient"), // Окружение
    PLAYERS(1F, "audioSettings.players"), // Игроки
    BLOCKS(1F, "audioSettings.blocks"), // Блоки
    WEATHER(0.1F, "audioSettings.weather"); // Погода


    private final float defaultVolume;
    private final String guiText;

    SoundCategory(float defaultVolume, String guiText){
        this.defaultVolume = defaultVolume;
        this.guiText = guiText;
    }

    public float getDefaultVolume(){
        return defaultVolume;
    }

    public String getTranslateKey(){
        return guiText;
    }

}

File: tests/test-minecraft/src/main/kotlin/pize/tests/minecraft/game/options/SoundCategory.kt
package pize.tests.minecraft.game.options

enum class SoundCategory private constructor(// Погода
    val defaultVolume: Float, val translateKey: String
) {
    MASTER(1f, "audioSettings.master"),

    // Общая Громкость
    MUSIC(0.1f, "audioSettings.music"),

    // Музыка
    AMBIENT(1f, "audioSettings.ambient"),

    // Окружение
    PLAYERS(1f, "audioSettings.players"),

    // Игроки
    BLOCKS(1f, "audioSettings.blocks"),

    // Блоки
    WEATHER(0.1f, "audioSettings.weather")

}
